src/context-selection/section-filter-selector-bar-item/section-filter-selector-bar-item.js

```diff
@@ -18,6 +18,14 @@ export default function SectionFilterSelectorBarItem() {
 
     useOnDependentParamsChange(deselect)
 
+    useEffect(() => {
```

Comment:
This could have been done in the "TabbedSections"-component, but doing it here colocates the code with the rest of the selection, and we're able to preselect the first section in a much earlier stage.